src/Symfony/Component/HttpKernel/Tests/EventListener/RobotsTagListenerTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\EventListener;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\EventListener\RobotsTagListener;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * @internal
+ */
+class RobotsTagListenerTest extends TestCase
+{
+    /**
+     * @param string|null $expected
+     * @param Response    $response
+     * @param bool        $debug
+     *
+     * @dataProvider provideResponses
+     */
+    public function testInvoke(?string $expected, Response $response, bool $debug = true): void
+    {
+        $listener = new RobotsTagListener($debug);
+
+        /** @noinspection PhpParamsInspection */
```

Comment:
First time I see this in the code base.

Reply:
It's to tell PHPStorm to ignore the type warning, but I can remove it